luigi/worker.py

```diff
@@ -411,6 +412,8 @@ class worker(Config):
                                    config_path=dict(section='core', name='worker-ping-interval'))
     keep_alive = BoolParameter(default=False,
                                config_path=dict(section='core', name='worker-keep-alive'))
+    keep_alive_ttl = TimeDeltaParameter(default=datetime.timedelta(0),
```

Comment:
Feels a little odd to maintain the legacy config format here and allow both `core -> worker-keep-alive-ttl` and `worker -> keep_alive_ttl` for a brand new config arg.

I'm not against it. Just seems odd to allow. But I'll let @Tarrasch or @honnix chime in on that

Reply:
could this also be added to Worker arg documentation?

Reply:
I'm lost here. Is there another place I should define this config option?

Reply:
I was providing some historical context.

i.e.
```
[worker]
keep_alive = True
```
used to be required to be
```
[core]
worker-keep-alive: True
```

Then Config classes came along and now
```
class worker(Config):
    keep_alive = BoolParameter(default=False)
```
knows to look at `[worker] keep_alive` for its possible override value.

----

So what i was mentioning here is that it's odd to add a new Worker cfg parameter and allow it to be included in the legacy `[core] worker-whatever-param` format.

----

Hopefully, that was less unclear (kinda hard to clearly articulate) :)